floss/main.py

```diff
@@ -543,6 +542,60 @@ def main(argv=None) -> int:
 
     language = identify_language(sample=sample, static_strings=static_strings)
 
+    if language == Language.GO:
+        enhancedStaticStrings = extract_go_strings(sample=sample, min_length=args.min_length)
+        enhancedStaticStrings = list(enhancedStaticStrings)
+
+        # Calculate String Coverage
+
+        all_static_string_list = []
+
+        for string_obj in static_strings:
+            if string_obj.string.isprintable():
+                all_static_string_list.append(string_obj.string)
+
+        ct = 0
+                
+        for string in all_static_string_list:
+            found = False
+            for j in enhancedStaticStrings:
+                k = j.string
+                if k in string:
+                    found = True
+                    break
+            if (not found) and (len(string) >= args.min_length):
+                ct += 1
+
+        logger.info("Percentage of strings extracted: " + str(100 - ((ct) * 100 / len(all_static_string_list))) + "%")
```

Comment:
and how about rendering this in a metadata (table) with more or less content depending on the verbosity?